apiserver/authentication/jwt/doc.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2024 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+// Package jwt provides an authentication mechanism whereby a JAAS-registered
+// Juju controller can accept a JAAS JWT (JSON Web Token) as the means to
+// declare a user and their permissions. The JWT is used to authenticate Juju
+// login requests against the api endpoint, and raw http requests.
```

Comment:
From "The JWT..." onwards:

Create two sections: Authentication and Authorisation.